### ISSUE TITLE
Find a question in this method.

### ISSUE THREAD

**Issue by wtllll** [cesarhuret/docGPT#14]
It seems that using chat-gpt in word can't relate the questions in the context. For example, if I ask chat-gpt to answer more concretely about the last question I aksed. It seems that chat can't remember the question. Have you guys come across such a problem. Is there anyway that I can solve this problem?

**Comment by cesarhuret** [comment]
Yeah so unfortunately my server doesn't remember what you ask it and save it into a conversation, but that's probably something you could change up in the `server` folder. 
If you'd like it's also something I could implement in some time
